doc/project/development_stages.md

```diff
@@ -33,34 +33,34 @@ Once the specification has been shared with the OpenTitan audience and sufficien
 
 The next life stage is **Development**.
 The hardware IP is being developed in GitHub, the specification is converted to Markdown, and design and verification planning is underway.
-This is a long phase expected to last until a more formal review is requested for full completion signoff.
+This is a long phase expected to last until a more formal review is requested for full completion sign-off.
 When in Development phase, the stage tracking of the design and verification milestones are valid.
 See those sections that follow for details there.
-To exit this stage, a signoff process review must occur.
-See the section on signoff for details.
+To exit this stage, a sign-off review must occur.
+See the section on sign-off for details.
 
-The final life stage is **Signed Off**.
+The final life stage is **Signed-Off**.
 At this point, a design is frozen and not expected to be updated.
-There are exceptions if post-signoff bugs are found, in which case the stage returns to Development and the version number is not updated.
+There are exceptions if post-sign-off bugs are found, in which case the stage returns to Development and the version number is not updated.
 Feature requests towards a signed-off design requires review and approval by the Technical Committee.
 Once accepted, it results in creating a new version and return a design to the appropriate life stage, based upon the size of the change.
 See the _Versioning_ section of the document for more discussion.
-Signed off fully-functioning (read: not buggy) designs stay in the Signoff stage as an available complete IP, with an associated revision ID.
+Signed-off fully-functioning (read: not buggy) designs stay in the signed-off stage as an available complete IP, with an associated revision ID.
```

Comment:
I think this should say `designs stay in the "Signed-Off stage"...` so the capitalisation matches the actual name of the stage.

Reply:
good point, updated (with a slightly different wording).